### ISSUE TITLE
Accessibility: Bypass Blocks

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#211]
Hello, this issue of "bypass blocks" are coming up in the [SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc?hl=en-US) chrome extension for all pages. 

To repair:

Warning
Skip repeated content
 2.4.1 Bypass Blocks

A means of bypassing blocks of content that are repeated on multiple web pages must be provided.

One of the ways to ensure this is to have a link at the top of each page that goes directly to the main content area. (The link should be the first focusable control on the web page and be visible when it receives focus). Such a link is not present on the page. If you have not ensured this by providing this type of links or by other means, this criterion is failed. Another way of ensuring conformance is to provide a heading for each block of content that are repeated on multiple web pages.
